### PR TITLE
KEYCLOAK-7274 Hardcoded config in offline adapter installation scripts

### PR DESCRIPTION
**Pull request by vramik**

### CODE REVIEW

**Comment by vramik** [comment]
https://github.com/keycloak/keycloak/pull/5184

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ endif::[]
 
 You can provide an adapter config file in your WAR and change the auth-method to KEYCLOAK within web.xml.
 
-Alternatively, you don't have to modify your WAR at all and you can secure it via the {project_name} adapter subsystem configuration in `standalone.xml`.
+Alternatively, you don't have to modify your WAR at all and you can secure it via the {project_name} adapter subsystem configuration in configuration file (e.g. `standalone.xml`).
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: ...configuration in the configuration file, such as `standalone.xml`.

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 2]
```diff
@@ -102,31 +102,33 @@ This ZIP archive contains JBoss Modules specific to the {project_name} adapter.
 
 To configure the adapter subsystem if the server is not running execute:
 
+NOTE: To be able to perform substitution for command line properties it has to be set `resolve-parameter-values` to `true` in `./bin/jboss-cli.xml`
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: ...command line properties you must set...

**Review by matthewhelmke — DISMISSED** [review]

**Review by matthewhelmke — DISMISSED** [review]
Looks good to me. Thank you!

**Comment by vramik** [comment]
@matthewhelmke I've updated the PR because it was found out there is no need to edit jboss-cli.xml file. Can you pls check the changes again?

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 3]
```diff
@@ -102,6 +102,8 @@ This ZIP archive contains JBoss Modules specific to the {project_name} adapter.
 
 To configure the adapter subsystem if the server is not running execute:
 
+NOTE: Alternatively, you can specify `server.config` property when installing adapters in command line to install adapters into different config. E.g. `-Dserver.config=standalone-ha.xml`
+
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: NOTE: Alternatively, you can specify the `server.config` property while installing adapters from the command line to install adapters using a different config, for example: `-Dserver.config=standalone-ha.xml`.

File: securing_apps/topics/oidc/java/jboss-adapter.adoc [hunk 4]
```diff
@@ -18,7 +18,7 @@ endif::[]
 
 You can provide an adapter config file in your WAR and change the auth-method to KEYCLOAK within web.xml.
 
-Alternatively, you don't have to modify your WAR at all and you can secure it via the {project_name} adapter subsystem configuration in `standalone.xml`.
+Alternatively, you don't have to modify your WAR at all and you can secure it via the {project_name} adapter subsystem configuration in the configuration file such as `standalone.xml`.
 Both methods are described in this section.
```
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/jboss-adapter.adoc]: ...in the configuration file, such as...

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Simple language adjustments

**Comment by vramik** [comment]
@matthewhelmke thank you

**Review by matthewhelmke — APPROVED** [review]

**Review by stianst — APPROVED** [review]

**Pull request merged**